bcda/auth/client/ssas_test.go

```diff
@@ -277,6 +278,54 @@ func (s *SSASClientTestSuite) TestGetToken() {
 	assert.Equal(s.T(), tokenString, string(respKey))
 }
 
+func (s *SSASClientTestSuite) TestGetVersionPassing() {
```

Comment:
I wonder if this can be written as a table driven test.

Your test struct could have the following:

- Response to return
- Expected version
- Expected error

That way the server set up and env var stuff is taken care of for you.